### ISSUE TITLE
Checkout Mail - Previous Owners

### ISSUE THREAD

**Issue by haupt-pascal** [grokability/snipe-it#10037]
**Server:**
 - SnipeIT, 5.0.11
 - OS: [Ubuntu]
 - Web Server: [Apache]
 - PHP, 7.4.3 

**Is your feature request related to a problem? Please describe.**
We've got the problem, that one part of the receivers from the checkout-mail wants to know the previous owners (last 2 or 3) of the asset. 

**Describe the solution you'd like**
Is there a way, and if not is it possbile, to add the previous owners of an asset into the checkout mail? 

**Additional context**
Let me know if you know a solution :)

**Comment by snipe** [comment]
Can you explain a little further? In the checkout notification email, you want to include the user it used to be checked out to? What's the use case here?

**Comment by haupt-pascal** [comment]
Nah, I want to include the user before the actual user - like a little history, you know what I mean?

**Comment by snipe** [comment]
I think that would likely expose information that admins wouldn't necessarily want to share with end-users in most use-cases.
